include/deployer.bash

```diff
@@ -76,10 +76,15 @@ cbd-version() {
 }
 
 cbd-update() {
-    declare desc="Binary selfupdater. Either latest github release (default), or specific branch from CircleCI"
+    declare desc="Binary selfupdater. Either latest github release"
 
     if [[ "$1" ]]; then
-        cbd-update-snap $1
+        # cbd-update-snap $1
+        warn "!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+        warn "This version is unsupported"
```

Comment:
maybe its better to inform that: "cbd update doesnt support any parameter"
please use "cbd update" to get the latest official release or ... (remaining text about contacting support)